krux/stats.py

```diff
@@ -40,42 +40,45 @@ def get_stats(
 
     :argument prefix: The string prepended to all stats sent to Statsd.
 
-    :keyword client: The StatsClient class to use. Defaults to 'True' which
-    signals the use of 'kruxstatsd.StatsClient'. 'False' or 'None' will
-    signal the use of :py:class:`stats.DummyStatsClient
-    <krux.stats.DummyStatsClient>`.  Any other value is interpreted as a
-    custom Stats class.
+    :parameter client: The StatsClient class to use. Defaults to 'True' which
+    signals the use of 'statsd.StatsClient'. 'False' or 'None' will
+    signal the use of :py:class:`stats.DummyStatsClient <krux.stats.DummyStatsClient>`.
+    `legacy` signals the use of kruxstatsd.StatsClient. Any other value is invalid.
```

Comment:
Is this backward compatible?
* If we were using the custom Stats class, we won't be able to accept this version and forward.
* If we were using `True`, then we will be getting the `statsd` version. Is that backward compatible?

Reply:
For the docstrings, I personally have been using following pattern:
```python
"""
Does magic

:param foo: Foo!
:type foo: str
:return: The result of magic
:rtype: bool
"""
```

Reply:
I changed the docstrings because my IDE flagged some params as not having docstrings (apparently because they were keywords).

Reply:
And yes, in the past it was possible to pass in a custom class instead of True/False etc. I did an audit of our usages, and I don't think anyone has ever done that. 

In place of that, I can add a `legacy_names` or similar kwarg, if the consensus is that is preferable.